### ISSUE TITLE
bigquery metrics dashboard in velodrome.k8s.io is out of date

### ISSUE THREAD

**Issue by spiffxp** [kubernetes/test-infra#8703]
Compare dashboards [bigquery metrics](http://velodrome.k8s.io/dashboard/db/bigquery-metrics?orgId=1) to [monitoring](http://velodrome.k8s.io/dashboard/db/monitoring?orgId=1)

Looks as though no data has shown up in a week.  I'm guessing this means the job responsible for populating it is hung or not running.

/area metrics
/area velodrome
/kind bug
/milestone 1.12
/sig testing
/meow space

**Comment by spiffxp** [comment]
related: https://storage.googleapis.com/k8s-gubernator/triage/index.html is also out of date

**Comment by AishSundar** [comment]
Kettle job was restarted and Velodrome has data now.

**Comment by spiffxp** [comment]
yay, backfill worked! I saw a doc float around with instructions on how to notice this / restart the job if it happens again, any reason we shouldn't have that checked into the repo?

**Comment by AishSundar** [comment]
This is [the PR](https://github.com/kubernetes/test-infra/pull/8750) to convert the doc into markdown

**Comment by spiffxp** [comment]
The linked doc refers to this as the issue for root-causing.  I'd rather open a new issue for that, and call this closed once the doc is merged

**Comment by spiffxp** [comment]
/close
root cause tracking issue https://github.com/kubernetes/test-infra/issues/8800
